hail/python/hail/backend/service_backend.py

```diff
@@ -202,7 +203,9 @@ async def create(*,
                 "MY_BILLING_PROJECT'"
             )
 
-        async_fs = RouterAsyncFS('file')
+        if not isinstance(gcs_requester_pays_configuration, str):
```

Comment:
It can also be `None`, right?

Reply:
right

Reply:
I was initially going to throw this into another PR and then stack this on top. Do you want me to just keep it in here though?

Reply:
Hmm. Actually it looks like this is still failing service backend tests that explicitly request a handful of requester pays buckets, e.g.
```
test_requester_pays_with_project_more_than_one_partition
```

Reply:
Ah, OK, I follow what your point is. You'll need to change RouterAsyncFS to intelligently use or not use the project based on the buckets and you hoped to do that in a follow up PR?

I think, unfortunately, we have to pull that into this one because folks, at least in principle, could be relying on that behavior.